src/Installers/Rpm/TargetingPack/Rpm.TargetingPack.rpmproj

```diff
@@ -27,6 +27,7 @@
     </ProjectReference>
 
     <InstallerOwnedDirectory Include="$(RpmPackageInstallRoot)packs/$(TargetingPackName)/" />
+    <RpmDependency Include="dotnet-targeting-pack-$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)" Version="$(MicrosoftNETCoreAppPackageVersion.Split('-')[0])" />
```

Comment:
Why is the package version different between debian and rpm installers? `DotnetRuntimeDependencyVersion` vs `MicrosoftNETCoreAppPackageVersion.Split('-')[0]`?

Reply:
Honestly, I just mirrored what was done in runtime installers but results of both operations would be the same. Also, Debian case uses token replacement and doesn't allow `Split`

Reply:
Feels strange that they are slightly different but okay.